src/DIRAC/Interfaces/scripts/dinit.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
```

Comment:
I'm not sure we need a third way to start using DIRAC

Reply:
You don't like the dinit ? As far as I understand this relates to the the caching mechanism @atsareg was so keen about.

Reply:
Though I somewhat agree that we should probably take out the proxy init stuff etc. But could I please ask for clemency and file this as an issue and I promise to fix it in pass 2 ?

Reply:
> But could I please ask for clemency and file this as an issue and I promise to fix it in pass 2 ?

If it was targetting integration maybe but we shouldn't be randomly adding and removing stuff from production branches.

Reply:
It's always been there fore the COMDIRAC users, so here we just do a 1:1 transfer of functionality. We can the deprecate it in integration.

Reply:
It's never been there for non-COMDIRAC users

Reply:
@fstagni Could I please ask for a manager level override for this issue ? I would like to keep this for the time being for backwards compatibility and once it gets used a bit more we can streamline it.

Reply:
Let's see who's more convincing at the coming BiLD meeting.